tests/components/dlna_dmr/test_media_player.py

```diff
@@ -31,7 +31,10 @@
 )
 from homeassistant.components.dlna_dmr.data import EventListenAddr
 from homeassistant.components.media_player import ATTR_TO_PROPERTY, const as mp_const
-from homeassistant.components.media_player.const import DOMAIN as MP_DOMAIN
+from homeassistant.components.media_player.const import (
+    DOMAIN as MP_DOMAIN,
+    MediaPlayerState,
+)
```

Comment:
The imports seem to be wrong. Please also check `mp_const`. I guess I wouldn't be worth the effort to replace the `ATTR_*` and `SERVICE_*` constants, but (almost) all other should have StrEnums.